### PR TITLE
Add minutes from 2020-06-04 meeting

### PR DESCRIPTION
**Pull request by reszelaz**
Hi all,
Here you can find minutes from today's meeting. As always feel free to modify/comment them.
Thanks a lot for your participation!

### CODE REVIEW

File: 20200604/MINUTES.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Minutes from the Sardana Follow-up Meeting - 2020/06/04
+
+Held on 2020/06/04 at 14:00
+
+Participants: ALBA - Zbigniew Reszela, DESY - Teresa Núñez, MAXIV - Aureo Freitas
+
+1. Demo:
+   * New MeasuremengGroup configuration API and expconf macros
+   * Improved PCTV widget and Taurus entry point for TauruForm items factories
+       * MaxIV integrates detectors as counter/timer controllers. The detecotrs
```
Comment (teresanunez) [inline 20200604/MINUTES.md]: Typo: detecotrs -> detectors

File: 20200604/MINUTES.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+2. Urgent user problems/issues - Round table
+    - DESY
+        - test Sardana Py3 with dummies and real hardware - OK for the moment
+            - to be on the safe side better use Tango 9.3.4rc5
+        - user's complain about dead time in scans
+    - MaxIV
+        - Continuous Scans integration with Eiger done - now writing unit tests
+            - [ ] Aureo will register the controllers in the sardana-plugin catalogue
+        - Continuous Scans integration for a BL at the end of the linac (*a kind of FEL*)
+            - machine timing system injection (2Hz - 10Hz) signals together with delay generaor are used to synchronize the detectors.
```
Comment (teresanunez) [inline 20200604/MINUTES.md]: Typo: generaor -> generator

**Review by teresanunez — APPROVED** [review]
Two typos commented

**Review by aureocarneiro — APPROVED** [review]

**Comment by reszelaz** [comment]
Thanks for the corrections! Merging!

**Pull request merged**